README.md

```diff
@@ -52,12 +52,19 @@ StakaterPlatform consist of 7 stacks:
 4. Ensure that correct context is set for kubectl & helm.
 5. run `make configure`, this will make all required substitutions based on configuration variables in the repository. When prompted commit those changes, don't commit them if you want to run everything from your local machine i.e. No flux hence no GitOps.
 6. For deployment there are two options:
+    
     a. Using [flux](https://playbook.stakater.com/content/processes/gitops/gitops-with-flux.html), [Add the public SSH key](https://help.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account) of flux(configs/flux.pub) to your Git repository with **write access**. 
+    
     b. Without flux, just don't add flux key and this will remove the process of GitOps(using flux) from your cluster. It removes the need of having your repository available remotely
 7. Run `make deploy` 
+    
     a. Using flux, this will deploy flux which will in turn deploy StakaterPlatform via GitOps.
+    
     b. If you don't want to use flux, just run `make deploy-without-flux` which will run `kubectl apply -f platform/` and deploy StakaterPlatform.
-9. Estimated time for everything to be up and running is 5-10. Use the printed dashboard token to access the Kubernetes dashboard at `dashboard-control.YOURDOMAINNAME`
+9. Estimated time for everything to be up and running is 5-10. Use the following command to get token to access kubernetes dashboard at `dashboard-control.YOURDOMAINNAME`:
+    ```
+    kubectl -n control describe secret $(kubectl -n control get secret | grep stakater-control-dashboard-kubernetes-dashboard-token | awk '{print $1}') | grep 'token:' | awk '{print $2}'
```

Comment:
You can add this and retain that it's printed in pipeline as well. Because we are going to print sealed-secrets keys in pipeline as well. Going forward we will try to find a better way of handling such variables